ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-release-4.6-presubmits.yaml

```diff
@@ -899,6 +899,72 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-vsphere,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - release-4.6
+    cluster: vsphere
+    context: ci/prow/e2e-vsphere-upgrade
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-machine-config-operator-release-4.6-e2e-vsphere-upgrade
+    rerun_command: /test e2e-vsphere-upgrade
```

Comment:
add optional: true

Reply:
also wondering why there is no 4.6 dry run below in prow ? :thinking:

Reply:
@kikisdeliveryservice its there it just succeeded :D

Reply:
ahhhh hahaha my bad :laughing: